deep_speech_2/README.md

```diff
@@ -283,28 +283,36 @@ The hyper-parameters $\alpha$ (coefficient for language model scorer) and $\beta
     CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \
     python tools/tune.py \
     --trainer_count 8 \
-    --alpha_from 0.1 \
-    --alpha_to 0.36 \
-    --num_alphas 14 \
-    --beta_from 0.05 \
-    --beta_to 1.0 \
-    --num_betas 20
+    --alpha_from 1.0 \
+    --alpha_to 3.2 \
+    --num_alphas 45 \
+    --beta_from 0.1 \
+    --beta_to 0.45 \
+    --num_betas 8
     ```
 
 - Tuning with CPU:
 
     ```bash
     python tools/tune.py --use_gpu False
     ```
+ The grid search will log the WER (word error rate) or CER (character error rate) at each point in the hyper-parameter space and their minima, and draw the error surface optionally. A proper hyper-parameters range should include the global minima of the error surface for WER/CER, as illustrated in the following figure.  
 
-After tuning, you can reset $\alpha$ and $\beta$ in the inference and evaluation modules to see if they really help improve the ASR performance.
+<p align="center">
+<img src="docs/images/tuning_error_surface.png" width=450>
+<br/>An example error surface for tuning on the dev-clean set of LibriSpeech
+</p>
+
+Usually, as the figure shows the variation of language model weight ($alpha$) mainly affect the performance of CTC beam search decoder. And a better procedure is first tuning on serveral data batches (the number can be specified) to find out the proper range of hyper-parameters, then change to the whole validataion set to carray out an accurate tuning.
```

Comment:
`as the figure shows` --> `as the figure shows, `
 `is first tuning` --> `is to first tune`

Reply:
Done